finger/forms.py

```diff
@@ -14,6 +15,22 @@ def clean_file(self):
         except:
             raise ValidationError("Must be proper json", code='badjson')
 
+class MailMembersForm(forms.Form):
+    subject = forms.CharField(label='Subject')
+    message = forms.CharField(label='Message', widget=forms.Textarea)
+    cc = forms.CharField(label='CC', required=False)
+    recipients = forms.CharField(label='Recipients', required=False)
```

Comment:
It should probably be spelled out how this field works, that an empty field equals "email everybody" sounds like an possible source of and error.

Reply:
Man kan inte "råka'" mejla alla eftersom när man trycker på knappen som gör ett testutskick så får man en lista på de som kommer att mejlas till.

Reply:
Varför betyder det att man inte kan råka mejla alla? Går det inte att skicka ett mail till alla eftersom det finns en testfunktion? Eller förlitar vi oss på att folk är rimliga och har sunt förnuft? Det är en sak jag slutade tro om användare på webben för ganska länge sedan, även om dessa råkar vara Stackens styrelse. :stuck_out_tongue: